website/docs/en/api/runtime-api/module-methods.mdx

```diff
@@ -360,6 +360,16 @@ if (module.hot) {
 }
 ```
 
+### require.resolveWeak
+
+<ApiMeta specific={['Rspack', 'Webpack']} />
+
+Similar to `require.resolve`, but this won't pull the module into the bundle. It's what is considered a "weak" dependency.
+
+```js
+require.resolveWeak(dependency: string);
+```
```

Comment:
Can we add the same example as webpack: https://webpack.js.org/api/module-methods/#requireresolveweak